examples/client/wayland_client.c

```diff
@@ -482,6 +488,70 @@ static struct xdg_toplevel_listener const shell_toplevel_listener =
     handle_xdg_toplevel__capabilities,
 };
 
+void run(struct wl_display* display, int* shutdown_signal, int* draw_signal, draw_context* ctx)
```

Comment:
As [g_]shutdown_signal and [g_]draw_signal are global, why pass them as parameters?

Reply:
This was my minor effort to make it less global-ly. But that might be a losing battle, I'm okay with using globals